windows/Dockerfile

```diff
@@ -0,0 +1,87 @@
+ARG UBUNTU_VERSION=20.04
```

Comment:
Does this need to be parameterized?

Reply:
It is in the challenge image. If we match that ubuntu version, you don't have to pull two different ubuntu images.

Reply:
This image is just used for building windows though, right? I think the reason we parameterized it on the challenge image is to support difference ubuntu bases for challenge runtimes (e.g. a 20.04 libc vs a 22.04 libc). If this is just a build helper, I don't think it makes sense to parameterize it.

Reply:
True, I don't want bumping the challenge ubuntu version to mysteriously break the windows builder either.